app/views/DR/EpidemiologicalResponseScreen/index.js

```diff
@@ -0,0 +1,44 @@
+import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { View } from 'react-native';
+
+import NavigationBarWrapper from '../../../components/NavigationBarWrapper';
+import Colors from '../../../constants/colors';
+import languages from '../../../locales/languages';
+import EpidemiologicalStatus from './epidemiologicalStatus';
+import MentalHealthAdvices from './mentalHealthAdvices';
+
+const TopBar = createMaterialTopTabNavigator();
+
+const EpidemiologicScreen = ({ navigation }) => {
+  const { t } = useTranslation();
+
+  return (
+    <NavigationBarWrapper
+      title={t('label.epidemiologic_report_title')}
+      onBackPress={() => navigation.goBack()}>
+      <View style={{ flex: 1, backgroundColor: Colors.WHITE }}>
+        <TopBar.Navigator
+          tabBarOptions={{
+            activeTintColor: '#0059ff',
+            inactiveTintColor: '#000',
```

Comment:
Could we use the color for the palette color file?

Reply:
Speak to @RutsSantos about that...